Library/Homebrew/diagnostic.rb

```diff
@@ -537,7 +537,7 @@ def check_casktap_integrity
         core_cask_tap = CoreCaskTap.instance
         return unless core_cask_tap.installed?
 
-        broken_tap(core_cask_tap) || examine_git_origin(core_cask_tap.git_repo, core_cask_tap.remote)
+        broken_tap(core_cask_tap) || examine_git_origin(core_cask_tap.git_repo, T.must(core_cask_tap.remote))
```

Comment:
What changed to make this necessary?

Reply:
Actually returning `super` in `CoreTap#remote` when `HOMEBREW_NO_INSTALL_FROM_API` is set.